Documentation/network/external-workloads.rst

```diff
@@ -72,23 +72,8 @@ clustermesh enable`` after ``cilium install``:
 
 .. code-block:: shell-session
 
-    cilium install --config tunnel-protocol=vxlan
-    cilium clustermesh enable
-
-Config option ``tunnel-protocol=vxlan`` overrides any default that could
-otherwise be auto-detected for your k8s cluster. This is currently a
-requirement for external workload support.
```

Comment:
Should this be preserved?

Reply:
yeah we can preserve it, i'll slightly rephrase it based on: https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/cilium-configmap.yaml#L439

> The ``tunnel=vxlan`` Helm value sets `tunnel-protocol` Cilium agent configuration option to `vxlan`. This is currently a requirement for external workload support.

Reply:
updated 📝